src/Codec/FFmpeg/Common.hsc

```diff
@@ -228,3 +278,30 @@ stringError err =
     peekCString buf
   where
     len = 1000
+
```

Comment:
Let's add a comment here. This is a very useful function that I think could be used in other places, too. We don't need to add all the possible uses now, but documenting what it is for will help with making those changes in the future.